app/controllers/api/v0/forem_directories_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V0
+    class ForemDirectoriesController < ApiController
+      def index
+        render json: {
+          cover_image_url: Settings::General.main_social_image,
+          description: Settings::Community.community_description,
+          logo_image_url: Settings::General.logo_png,
+          name: Settings::Community.community_name,
+          tagline: Settings::Community.tagline,
+          version: "edge.#{Time.current.strftime('%Y%m%d')}.0",
```

Comment:
This is hardcoded to mimic our plan for versioning.

Reply:
**Question (non-blocking):** Could it make sense to use `Time.now.utc` explicitly here? `Time.current` can change depending on `Time.zone` or `config.time_zone` and while this is unlikely using the most explicit version with the fewest possible surprises seems like a potentially good idea to me.

Reply:
@citizen428 IIRC Rubocop complains if we use `Time.now` 😄

@atsmith813 I didn't notice anything in this PR that uses this value, but do you have (or know of) anything planned that will? Because this points out another thing I overlooked with versioning, which is how we'd get that value into the code. Could we chat about this a bit outside the context of this PR? Maybe in Slack or forem.{team,dev}?